app/assets/javascripts/custom.js

```diff
@@ -109,4 +106,81 @@ $( "#invite" ).click(function() {
   $("#invite").attr("disabled", true);
 })
 
+function openNav() {
+      var e = document.getElementById("mySidenav");
+      if (e.style.width == '250px')
+      {
+          e.style.width = '0px';
+      }
+      else 
+      {
+          e.style.width = '250px';
+      }
+  }
```

Comment:
One cleaner way to do this, is to have a class that has width 250px, and add or remove that class as required. Setting the width in JS usually makes it difficult to maintain later on.